android/java/org/chromium/chrome/browser/crypto_wallet/util/AssetUtils.java

```diff
@@ -56,4 +57,21 @@ public static boolean isAuroraAddress(String contractAddress, String chainId) {
         return (isEthereumBridgeAddress || isNativeAsset)
                 && chainId.equals(BraveWalletConstants.MAINNET_CHAIN_ID);
     }
+
+    public static @CoinType.EnumType int getCoinForKeyring(String keyringId) {
+        int coin = CoinType.ETH; // For default keyring
+        switch (keyringId) {
+            case BraveWalletConstants.SOLANA_KEYRING_ID:
```

Comment:
do we need to show an account creation for ETH Dapps? Or is it applicable for SOL only?

Reply:
It's just a default value. We will never have to show ETH account creation, if it ever happens then it will be significant bug either on native side or in the usage of invalid dapps support for coin on android. For time being, it's only for Solana.